dags/gcs_bucket.py

```diff
@@ -24,3 +24,4 @@
 # GCS bucket for output
 BENCHMARK_OUTPUT_DIR = "gs://ml-auto-solutions/output/benchmark"
 XLML_OUTPUT_DIR = "gs://ml-auto-solutions/output/xlml"
+MAXTEXT_OUTPUT_DIR = "gs://ml-auto-solutions/output/maxtext/nightly"
```

Comment:
Will it be better we have `gs://ml-auto-solutions/output/maxtext` here? So your team could use it for nightly and other versions?

Reply:
We usually test `nightly` and `stable`

Maybe it will be good to have 

`gs://ml-auto-solutions/output/maxtext/nightly`
and 
`gs://ml-auto-solutions/output/maxtext/stable`

So the results can be more clear, what do you think?

Reply:
Will use `gs://ml-auto-solutions/output/maxtext` for MAXTEXT_OUTPUT_DIR and add `nightly` in here:
https://github.com/GoogleCloudPlatform/ml-auto-solutions/pull/77/files#diff-c697bd6d7e4d4647f9a1e3e094ded8e445df4a82583823b7280e8f90154e3925R56